iframes/iframe-init.ts

```diff
@@ -4,7 +4,9 @@
  */
 
 import { Datex } from "../datex.ts";
-import "./iframe-com-interface.ts";
+import { communicationHub } from "../network/communication-hub.ts";
+import { WindowInterface } from "../network/communication-interfaces/window-interface.ts";
 
-await Datex.Supranet.connect();
-await Datex.InterfaceManager.connect("iframe", undefined, [parent])
\ No newline at end of file
+await Datex.Supranet.init();
+const windowInterface = WindowInterface.createParentInterface(window.parent)
+await communicationHub.addInterface(windowInterface, true)
```

Comment:
We probably shouldn't use the iframe interface as default interface. If theres a direct if we should prefer this one.